treeshr/treeshrp.h

```diff
@@ -632,7 +632,8 @@ typedef struct pino_database
   unsigned modified : 1;      /* Flag indicating tree structure modified */
   unsigned setup_info : 1;    /* Flag indicating setup info is being added */
   unsigned remote : 1;        /* Flag indicating tree is on remote system */
-  unsigned advanced_compression : 1;
+  unsigned alternate_compression : 1;
+  unsigned char : 1;
```

Comment:
The type for the flags is `unsigned`, not `unsigned char` so this should be `unsigned : 25`, and I would check the `sizeof` before and after to verify we didn't accidentally inflate it

Reply:
Well - 
  Some of the bitfields in this file are `unsigned name : width` and some are `unsigned char name: width`.  I changed them all to the former.

Reply:
Those are completely different things, you can't change one to the other. `unsigned char` is 1 byte and `unsigned` is shorthand for `unsigned int` which is 4 bytes